### PR TITLE
task 11 is complete

### PR DESCRIPTION
**Pull request by sodikabdullaev**

### CODE REVIEW

File: models/articles.model.js [hunk 1]
```diff
@@ -13,21 +14,24 @@ exports.selectArticleById = (id) => {
 			else return rows
 		})
 }
-exports.selectArticles = () => {
-	return db
-		.query(
-			`SELECT articles.author, articles.title, articles.article_id, 
+exports.selectArticles = (topic) => {
+	const values = []
+	const conditions = []
```
Comment (nc-mezz-davies) [inline models/articles.model.js:19]: What is this for? It is not being used. 🙂

**Review by nc-mezz-davies — COMMENTED** [review]
Looking good but what happens if we make a **valid** topic query request for a topic that has no articles? Similar to an article that has no comments this is not an error. Have a look in the test data if you are a bit lost and then make a test for this please

File: models/articles.model.js (hunk 1 above)
Comment (sodikabdullaev) [reply models/articles.model.js:19]: ohh that makes sense, those topics filter comes from topics list, needs to be verified from topics table right.

Comment (sodikabdullaev) [reply models/articles.model.js:19]: yeah conditions var is not being used,I 
 will delete it

**Pull request merged**